metadata/metadata.go

```diff
@@ -433,6 +442,10 @@ func (resource *featureResource) ID() ResourceID {
 	}
 }
 
+func (resource *featureResource) Schedule() string {
+	return ""
```

Comment:
Type "" noSchedule. Unclear what this does with just a empty string

Reply:
or should I panic? since schedule isn't implemented for this?